### PR TITLE
Feature: unit test for slack team controller class

### PR DESCRIPTION
**Pull request by Danijel-Kojic**

### CODE REVIEW

File: server/src/modules/slack/team/controllers/slack.team.controller.spec.ts [hunk 1]
```diff
@@ -12,11 +15,39 @@ describe('SlackTeamController', () => {
...
-  describe('root', () => {
-    it('should return "Hello World!"', () => {
-      expect(slackTeamController.getAccessLogs()).toBe('Hello World!');
+  describe('getAccessLogs', () => {
+    it('should get access logs and set response heades', async () => {
+      // Mocking dependencies and setting up test data
+      const authHeader = "Bearer token xxx";
+      const maxPage = 1;
+      const response = {
+        setHeader: jest.fn(),
```
Comment (Binary-Vanguard-12138) [inline server/src/modules/slack/team/controllers/slack.team.controller.spec.ts:27]: What does this line mean?
Why do we need to mock setHeader and status functions?
Are you sure?

File: server/src/modules/slack/team/controllers/slack.team.controller.spec.ts [hunk 2]
```diff
@@ -12,11 +15,39 @@ describe('SlackTeamController', () => {
...
+        status: jest.fn(),
+      };
+      const stream = new Readable();
+      const mockServiceResponse = "Mocked access logs";
+      jest.spyOn(slackTeamService, "getAccessLogs").mockImplementation(async (authHeader, maxPage, stream) => {
```
Comment (Binary-Vanguard-12138) [inline server/src/modules/slack/team/controllers/slack.team.controller.spec.ts:32]: Does it actually work?
Are you sure?

**Review by David-Hang-12138 — COMMENTED** [review]
1 unit tests fails.

**Review by David-Hang-12138 — CHANGES_REQUESTED** [review]
We need to make unit tests working.

**Review by David-Hang-12138 — APPROVED** [review]
I think this PR is good.

**Pull request closed without merging**